src/Websites/Websites.Autorest/custom/New-AzStaticWebApp.ps1

```diff
@@ -285,41 +285,15 @@ param(
     ${ProxyUseDefaultCredentials}
 )
 
-begin {
-    try {
-        $outBuffer = $null
-        if ($PSBoundParameters.TryGetValue('OutBuffer', [ref]$outBuffer)) {
-            $PSBoundParameters['OutBuffer'] = 1
+    process {
+        try {
+            if(!$PSBoundParameters.ContainsKey('RepositoryUrl')) {
+                $PSBoundParameters.RepositoryUrl = $null
```

Comment:
```suggestion
                $PSBoundParameters.RepositoryUrl =''
```

Reply:
Done, and update some examples file, because these examples have extra lines or spaces